doc/en/api/1.0.0/coveragestores.yaml

```diff
@@ -359,6 +349,72 @@ paths:
         405:
           description: Method Not Allowed
 
+parameters:
+  CoverageStorePost:
+    name: coverageStoreBody
+    description: |
+      The coverage store body information to upload.
+
+      Examples:
+      - application/xml:
+
+        ```
+        <coverageStore>
+          <name>nyc</name>
+          <url>file:/path/to/file.tiff</url>
+        </coverageStore>
+        ```
+
+      - application/json:
+
+        ```
+        {
+          "coverageStore": {
+            "name": "nyc",
+            "url": "file:/path/to/file.tiff"
+          }
+        }
+        ```
+
+
+    in: body
+    required: true          
+    schema:
+      $ref: "#/definitions/CoverageStoreInfo"
+  CoverageStorePut:
+    name: coverageStoreBody
+    description: |
+      The coverage store body information to upload.
+      For a PUT, only values which should be changed need to be included.
+
+      Examples:
+      - application/xml:
+
+        ```
+        <coverageStore>
+          <description>A coverage store</description>
+          <enabled>true</enabled>
+          <__default>true</__default>
+          <url>file:/path/to/file.tiff</url>
+        </coverageStore>
+        ```
+
+      - application/json:
+
+        ```
+        {
+          "coverageStore": {
+            "description": "A coverage store",
+            "enabled": "true",
+            "_default": "true",
+            "url": "file:/path/to/file.tiff"
+          }
+        }
+        ```
```

Comment:
This looks good covering the 90% case.

It can be left to the docs for individual formats to explore relative paths, path variables or external URLs.